src/main.cpp

```diff
@@ -111,9 +112,8 @@ int main(int argc, char** argv) {
         ("hivecleaner",
          po::value<bool>(&hiveCleaner)->default_value(hiveCleaner),
          "enable or disable hiveCleaner")
-        ("index",
-         po::value<std::string>(&elastic_index)->default_value(elastic_index),
-         "Elastic index to add the data to.")
+        ("index", po::value<std::string>(&elastic_index)->default_value(elastic_index), "Elastic index to add the data to.")
+        ("featurestore_index", po::value<std::string>(&elastic_featurestore_index)->default_value(elastic_featurestore_index), "Elastic featurestore index.")
```

Comment:
you will need to add featurestore_index to the config.template and epipe-chef

Reply:
fixed